src/lib/mongoBackend/mongoUpdateContext.cpp

```diff
@@ -68,16 +68,26 @@ HttpStatusCode mongoUpdateContext
     else
     {
         /* Process each ContextElement */
-        for (unsigned int ix= 0; ix < requestP->contextElementVector.size(); ++ix)
+        for (unsigned int ix = 0; ix < requestP->contextElementVector.size(); ++ix)
         {
-          processContextElement(requestP->contextElementVector.get(ix),
-                                responseP,
-                                requestP->updateActionType.get(),
-                                tenant,
-                                servicePathV,
-                                uriParams,
-                                xauthToken,
-                                caller);
+          //
+          // For an appendonly operation, there is only one ContextElement.
+          // If this ContextElement is already existing in mongo, processContextElement
+          // returns error 1
+          //
+          int r = processContextElement(requestP->contextElementVector.get(ix),
+                                        responseP,
+                                        requestP->updateActionType.get(),
+                                        tenant,
+                                        servicePathV,
+                                        uriParams,
+                                        xauthToken,
+                                        caller);
+          if ((r == 1) && (strcasecmp(requestP->updateActionType.get().c_str(), "appendonly") == 0))
```

Comment:
Is this logic compatible with a "direct" usage of the appendonly functionality in v1 standar operations? The `return SccNoContent` makes me wonder if this piece of code relies to much in the caller (v2) serviceRoutine.

But probalby this piece of code is going to change after our last chat at skype, so let's hold on this comment.